package.json

```diff
@@ -37,7 +37,7 @@
     "peer-id": "^0.14.0"
   },
   "peerDependencies": {
-    "ipfs-http-client": "^47.0.1"
+    "ipfs-http-client": "*"
```

Comment:
I get the need to make this less strict, but I think ranges are more appropriate here. https://docs.npmjs.com/cli/v7/using-npm/semver#ranges

By specifying a known minimum we still give the user some feedback in case they try to use with older versions of IPFS and whenever there is a true breaking change with this module we can bump this then.

So for this, we would have a range like `>=47.0.0` (or whatever the last valid major was).